src/js/app.js

```diff
@@ -3,9 +3,9 @@
  *
  * This is where you write your app.
  */
-
-var UI = require('ui');
-var Vector2 = require('vector2');
+require('./main');
+var UI = require('./ui');
+var Vector2 = require('./lib/vector2');
```

Comment:
I can add logic that checks `package.json` at runtime. `master` is more or less development, so it's okay to merge to master as long as the demo app works.

I'm wondering however, if you add Pebble.js as a package, the host app would need to do `require('pebblejs/ui')` right? If so, the check is only for those that build their app directly into the pebblejs repo which may become a deprecated use case.